keyboards/wings42/rev1_extkeys/readme.md

```diff
@@ -0,0 +1,19 @@
+# wings42
+
+![wings42](https://raw.githubusercontent.com/yfuku/wings42/main/images/wings42.jpg)
+
+A split keyboard with 3x6 vertically staggered keys and 3 thumb keys.
+
+* Keyboard Maintainer: [yfuku](https://github.com/yfuku)
+* Hardware Supported: wings42 PCB, Pro Micro
+* Hardware Availability: https://yfuku.booth.pm/
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
I'm sorry. I couldn't grasp the meaning. Is it insufficient as it is?

Reply:
For instance, is there a hardware reset button?  is bootmagic enabled? And fi so, what key is the key to jump to the bootloader?   location of reset keycode in default keymap?

Reply:
This is a keyboard with a plate for key expansion.
I'm sorry it's only in Japanese, but this is what it looks like.
https://yfuku.com/wings42/ext/keys/

Reply:
@drashna Added the url to the build guide.

Reply:
Thanks!

And no worries, most browsers do have translation support. Which, while isn't perfect, is usually "good enough" to understand.